luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -0,0 +1,240 @@
+        conseq_predicted_gene_cols = [
+            gene_col for gene_col in self.mt.info if gene_col.startswith(CONSEQ_PREDICTED_PREFIX)
+            and gene_col not in NON_GENE_PREDICTIONS
+        ]
+        mapped_genes = [
+            self.mt.info[gene_col].map(
+                lambda gene: hl.struct(**{
+                    'gene_symbol': gene,
+                    'gene_id': self.gene_id_mapping.get(gene, hl.missing(hl.tstr)),
+                    'major_consequence': gene_col.replace(CONSEQ_PREDICTED_PREFIX, '', 1)
+                })
+            )
+            for gene_col in conseq_predicted_gene_cols
+        ]
+        return hl.flatmap(
+            lambda x: x, 
+            hl.filter(
+                lambda x: hl.is_defined(x),
+                mapped_genes,
+            )
+        )
+
+    @row_annotation(fn_require=xpos)
+    def xstart(self):
+        return self.mt.xpos
+
+    @row_annotation(fn_require=end_locus)
+    def xstop(self):
+        return variant_id.get_expr_for_xpos(self.mt.end_locus)
+
+    # rg37_locus is annotated by HailMatrixTableTask.add_37_coordinates
+
+    @row_annotation(fn_require=end_locus)
+    def rg37_locus_end(self):
+        return hl.if_else(
+            self.mt.end_locus.position <= hl.literal(hl.get_reference('GRCh38').lengths)[self.mt.end_locus.contig],
+            hl.liftover(hl.locus(self.mt.end_locus.contig, self.mt.end_locus.position, reference_genome='GRCh38'), 'GRCh37'),
+            hl.missing('locus<GRCh37>')
+        )
+
+    @row_annotation(name='svType')
+    def sv_type(self):
+        return self.mt.alleles[1].replace('[<>]', '').split(':', 2)[0]
+
+    @row_annotation(name='transcriptConsequenceTerms', fn_require=[
+        sorted_transcript_consequences, sv_type,
+    ])
+    def transcript_consequence_terms(self):
+        return self.mt.sortedTranscriptConsequences.map(lambda x: x.major_consequence).extend([self.mt.svType])
+
+    @row_annotation()
+    def sv_type_detail(self):
+        sv_types = self.mt.alleles[1].replace('[<>]', '').split(':', 2)
+        return hl.if_else(
+            sv_types[0] == 'CPX',
+            self.mt.info.CPX_TYPE,
+            hl.if_else(
+                (sv_types[0] == 'INS') & (hl.len(sv_types) > 1),
+                sv_types[1],
+                hl.missing('str')
+            )
+        )
+
+    @row_annotation(name='geneIds', fn_require=sorted_transcript_consequences)
+    def gene_ids(self):
+        return hl.set(
+            hl.map(
+                lambda x: x.gene_id,
+                self.mt.sortedTranscriptConsequences.filter(
+                    lambda x: x.major_consequence != 'NEAREST_TSS'
+                )
+            )
+        )
+
+    @row_annotation(name='variantId')
+    def variant_id(self):
+        return self.mt.rsid
+
+    # NB: This is the "elasticsearch_mapping_id" used inside of export_table_to_elasticsearch.
+    @row_annotation(name='docId', disable_index=True)
+    def doc_id(self, max_length=512):
```

Comment:
I added this to make [this line](https://github.com/broadinstitute/seqr-loading-pipelines/blob/e2109adfc508112be0c613ad2efb68e884b861ce/luigi_pipeline/lib/hail_tasks.py#L256) work in the elastic search export.  But the field itself is not present in the `load_data` export.  Was this the right thing to do?